commands/info.md

```diff
@@ -300,8 +300,10 @@ Here is the meaning of all fields in the **stats** section:
 *   `total_writes_processed`: Total number of write events processed
 *   `io_threaded_reads_processed`: Number of read events processed by the main and I/O threads
 *   `io_threaded_writes_processed`: Number of write events processed by the main and I/O threads
-*   `stat_reply_buffer_shrinks`: Total number of output buffer shrinks
-*   `stat_reply_buffer_expands`: Total number of output buffer expands
+*   `client_query_buffer_limit_disconnections`: Total number of clients reached query buffer length limit
```

Comment:
how about "Total number of disconnections due to the client reached query buffer limit" and "Total number of disconnections due to the client reached output buffer limit"

I would like to emphasize the "disconnection".